components/button/button-move.js

```diff
@@ -49,11 +51,41 @@ class ButtonMove extends FocusMixin(RtlMixin(LitElement)) {
 			 * @type {boolean}
 			 */
 			disabled: { type: Boolean, reflect: true },
+			/**
+			 * Disables the down interaction
+			 * @type {boolean}
+			 */
+			disabledDown: { type: Boolean, attribute: 'disabled-down' },
```

Comment:
Wondering if you could just get away with `disabled-down` and `disabled-up` since those are the only things that are visually conveyed? Then the consumer could just ignore home/end/left/right if they consider them not possible options.

Reply:
Yeah, that's certainly an option. That's what I originally had, but I didn't want to leak the up/down events since the event handler would have to do some work to re-evaluate whether the operation is valid. It followed that I wanted to treat them all the same way. I don't have a current need to disable left/right/home/end, but as soon as that need arises, I would want to add the properties to stop the events. Thouhts?

Reply:
Could we make the assumption at least that "home" is tied to the "left" button and "end" is tied to the "right" one? And then should we be adding "page up" and "page down" support for the up/down equivalent?

If so, then you could suppress the "home" event when the right button is disabled, and vice versa.

Reply:
Hmmm... in the list's case, the home/end are more closely aligned with up/down. It's hard to say what home/end really mean in this context. But page-up/page-down could certainly come into play as well. 🤷

Reply:
Interesting, that's not what I would have expected as page up/down are what I'd normally use to "scroll all the way up" or down. Anyway that's getting into changing how drag & drop on lists works, which is out of scope here for sure!

Reply:
Yeah I don't recall how they arrived at that. There is some overlap with the grid mode of lists, where Home/End moves focus to the first/last items, and PageUp/PageDown moves the focus up/down by 5 items. I seem to recall Jeff also asking for PageUp/PageDown to do similar with moving items up/down by 5, but I don't think it made it into the code.